<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSActivateDevice</class>
 <widget class="QDialog" name="GBSActivateDevice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>14</y>
      <width>95</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;</string>
    </property>
    <property name="text">
     <string>输入客户号</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>2</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../forms/images/gbs/biz/gbs-close-window.png</normaloff>../../forms/images/gbs/biz/gbs-close-window.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>50</y>
     <width>831</width>
     <height>411</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="leDeviceName">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border: 1px solid  #7E7E7E;
border-radius: 8px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>  D01(必填)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leActivateCode">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>490</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border: 1px solid  #7E7E7E;
border-radius: 8px;
</string>
    </property>
    <property name="placeholderText">
     <string>  激活编号(必填)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>300</y>
      <width>96</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:10px;
font-size:20px;</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>300</y>
      <width>96</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:10px;
font-size:20px;</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;
color:#7E7E7E;</string>
    </property>
    <property name="text">
     <string>设备名称</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;
color:#7E7E7E;</string>
    </property>
    <property name="text">
     <string>激活编号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>541</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;
color:#7E7E7E;</string>
    </property>
    <property name="text">
     <string>请复制您的激活编号到场控管理添加授权或者给到您的代播运营者添加授权</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leLiveAccount">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>490</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border: 1px solid  #7E7E7E;
border-radius: 8px;
</string>
    </property>
    <property name="placeholderText">
     <string>  直播平台的账号(必填)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>140</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;
color:#7E7E7E;</string>
    </property>
    <property name="text">
     <string>平台账号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>260</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;
color:#7E7E7E;</string>
    </property>
    <property name="text">
     <string>远程账号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRemoteAccount">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border: 1px solid  #7E7E7E;
border-radius: 8px;
</string>
    </property>
    <property name="placeholderText">
     <string>  远程桌面账号(选填）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leRemotePassword">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>191</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border: 1px solid  #7E7E7E;
border-radius: 8px;
</string>
    </property>
    <property name="placeholderText">
     <string>  远程桌面密码(选填）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>260</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;
color:#7E7E7E;</string>
    </property>
    <property name="text">
     <string>远程密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCustom1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border: 1px solid  #7E7E7E;
border-radius: 8px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>  自定义1(选填）</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>200</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;
color:#7E7E7E;</string>
    </property>
    <property name="text">
     <string>用户备注</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>190</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_51">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>200</y>
      <width>66</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;
color:#7E7E7E;</string>
    </property>
    <property name="text">
     <string>直播平台：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leCustom2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border: 1px solid  #7E7E7E;
border-radius: 8px;
</string>
    </property>
    <property name="placeholderText">
     <string>  自定义2(选填）</string>
    </property>
   </widget>
   <widget class="EllipticalSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>23</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;
color:#7E7E7E;</string>
    </property>
    <property name="text">
     <string>远程开关</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>复制</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EllipticalSlider</class>
   <extends>QSlider</extends>
   <header location="global">gbs/common/EllipticalSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
